File: backend/CncApp/CncApp.Application/Dtos/StockLots/StockLotSummaryDto.cs
using CncApp.Domain.Enums;

namespace CncApp.Application.Dtos.StockLots;

public class StockLotSummaryDto
{
    public int Id { get; set; }
    public string LotNumber { get; set; } = string.Empty;
    public int AmountOfBars { get; set; }
    public decimal Diameter { get; set; }
    public decimal BarLength { get; set; }
    public StockLotConditionEnum Condition { get; set; }
    public DateTimeOffset CheckedInDateTime { get; set; }
}

File: frontend/angular/src/app/core/dtos/stock-lots/stock-lot-summary.dto.ts
import { StockLotCondition } from '../shipping-receiving';

export interface StockLotSummaryDto {
  id: number;
  lotNumber: string;
  amountOfBars: number;
  diameter: number;
  barLength: number;
  condition: StockLotCondition;
  checkedInDateTime: string;
}
